Form/Type/AddressEmbeddableType.php

```diff
@@ -53,6 +53,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             $countryCodeOptions['choice_loader'] = null;
         }
         $countryCodeOptions['preferred_choices'] = $options['preferred_countries'];
+        if (isset($options['choice_translation_domain'])) {
```

Comment:
See below, the isset can be avoided I think.

Reply:
Well, in the doc, the default value is TRUE, but in the code, it is FALSE, isn't it? https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Form/Extension/Core/Type/CountryType.php#L43

That's why i did that way